apps/wallet-mobile/src/features/SetupWallet/common/MnemonicInput/MnemonicInput.tsx

```diff
@@ -257,6 +257,7 @@ const MnemonicWordInput = React.forwardRef<RNTextInput, MnemonicWordInputProps>(
             errorDelay={0}
             errorText={error}
             autoComplete="off"
+            selectionColor="#D4D8E3"
```

Comment:
https://emurgo.atlassian.net/browse/YOMO-1350

Reply:
reverted `selectionColor ` changes caret and highlight

needs a separate investigation